### PR TITLE
Rabi content has been moved to ReCirq: Delete the items in Cirq

### PR DESCRIPTION
**Pull request by augustehirth**
Rabi Oscillations have been moved per quantumlib/ReCirq#292.
Also update the intro notebook to not use now-moved recirq experiment (replaced with single_qubit_state_tomography)
Also ran a formatter and cleared outputs from intro notebook

Only merge with redirect cl/452988155

### CODE REVIEW

**Comment by dabacon** [comment]
LGTM, there are some conflict with #5409 most importantly probably the picture (picture is not Sycamore, it's Bristlecone!), so whichever gets merged first will need to reconcile.

File: docs/tutorials/educators/intro.ipynb [hunk 1]
```diff
@@ -2,14 +2,14 @@
  "cells": [
```
Comment (dabacon) [inline docs/tutorials/educators/intro.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #15.    print("Before optimizing:\n{}\n".format(circuit))
</pre><p>slight pref for f-strings these days.  Probably should at least be consistent across the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='160'/>

**Review by dabacon — APPROVED** [review]
LGTM

**Comment by augustehirth** [comment]
Where is is it used internally, except for that intro page? Where would you like it to be used? 

My thought is that: A "hello world" level example shouldn't be so complicated that hiding away the circuit construction in a library call is necessary. If a function that generates a specific type of circuit is so widely useful, it may definitely be useful to bring it back in some form, perhaps in the same way as `cirq.qft`.

**Comment by maffoo** [comment]
In this case circuit is not particularly complicated to construct; the nice thing about the experiment function is the result type which can be plotted without parsing sweep results and converting to probs, and the fact that the whole thing can be run as a one-liner after just importing `cirq`, without needing to import sympy for symbols, matplotlib for plotting, etc.

The places where we use this internally are more like hardware hello world examples, that is, places where we're trying to demonstrate basic functioning of a real hardware device, rather than basics of cirq. We just want to take some actual data and plot it to see that it looks reasonable. I guess this is more like a hello world for `cirq.experiments` since rabi oscillations are one of the simplest things one can demonstrate on a quantum device (certainly simpler than the tomography experiments or xeb which are still here).

**Pull request merged**